packages/edit-site/src/components/sidebar-navigation-screen/style.scss

```diff
@@ -10,6 +10,19 @@
 	color: $gray-600;
 }
 
+.edit-site-sidebar-navigation-screen__page-link {
+	color: $gray-600;
+
+	&:hover,
+	&:focus {
+		color: $white;
+	}
+
+	.components-external-link__icon {
+		margin-left: $grid-unit-5;
```

Comment:
This won't build for me, I get:
```
'Error: Undefined variable.\n' +
    '   ╷\n' +
    '22 │         margin-left: $grid-unit-5;\n' +
    '   │                      ^^^^^^^^^^^^\n' +
    '   ╵\n' +
```

Reply:
Pushed a fix for this

Reply:
I'm surprised this isn't part of the component itself. It might be worth looking into a more general fix for the component, in a followup PR...

Reply:
> I'm surprised this isn't part of the component itself. It might be worth looking into a more general fix for the component, in a followup PR...

I think it's cause this can be used in smaller inline text, where its not necessary to add space before the icon. Although we probably should have some space after the icon in the screenshot: 

<img width="279" alt="CleanShot 2023-03-01 at 17 47 22" src="https://user-images.githubusercontent.com/1813435/222282979-3a2ce47d-f221-4981-8b00-4a754594bb41.png">

Reply:
> Pushed a fix for this

Did you push it?

Reply:
oops no, I have now!

Reply:
Weird — it both built and showed a space as if it was working for me 🤷🏻 Thanks for catching and fixing that!